packages/shared/src/server/ingestion/types.ts

```diff
@@ -119,6 +123,8 @@ export const CreateGenerationBody = CreateSpanBody.extend({
     )
     .nullish(),
   usage: usage,
+  usageDetails: UsageOrCostDetails,
```

Comment:
I am wondering whether we should add some sort of API versioning here. Then we could more easily track who is using old fields and remove them more safely.